components/dashboard/src/components/PrebuildLogs.tsx

```diff
@@ -57,6 +68,11 @@ export default function PrebuildLogs(props: PrebuildLogsProps) {
                             }
                             logsEmitter.emit("logs", content.text);
                         },
```

Comment:
This will get triggered by _any_ prebuild this user has access to.
Maybe a simple `if (update.info && update.info.buildWorkspaceId === props.workspaceId)` already is enough.